eyeshade/migrations/0010_geo_referral/up.sql

```diff
@@ -0,0 +1,25 @@
+select execute($$
+
+insert into migrations (id, description) values ('0010', 'geo_referral');
+
+create table geo_referral_groups(
+  id                    uuid primary key default uuid_generate_v4(),
+  created_at            timestamp with time zone not null default current_timestamp,
+  min_referral_time     timestamp with time zone not null default current_timestamp,
```

Comment:
What is `min_referral_time`?  Is it possible to rename it to something ending in `_at`?

Reply:
so, min referral time is the cutoff time for download timestamps that the promo server uses.
we could change this into active_at or something like that.